### ISSUE TITLE
Document hierarchy for HOT tools and projects

### ISSUE THREAD

**Issue by smit1678** [hotosm/design-strategy#3]
Outline the hierarchy for the ecosystem of HOT tools and projects.

**Comment by smit1678** [comment]
First take at outlining hierarchy. Core tools and sites either currently have or could have HOT brand emphasized. HOT-led or coordinated projects or initiatives are where priorities for specific project branding are emphasized. This is for wider community adoption or other project priorities. 

#### Core tools/sites:

- hotosm.org
- Country offices/intiatives
  - HOT ID
- Tasking Manager
- Export Tool
- Training Center

#### HOT-led/coordinated Projects or Initiatives:

- OpenAerialMap
- LearnOSM
- Field Papers (to be HOT-led)
- Collaborations
  - Ramani Huria
  - Missing Maps
  - MapUganda

@hotosm/design-strategy-working-group What are we missing or needs to be reorganized? 

cc @dhakelila

**Comment by PaulUithol** [comment]
TeachOSM? Not sure how that relation is actually. Although an area I'd also like to do more on. We have a lot of teaching materials around, just not always easily accessible at the moment.

YouthMappers?

**Comment by PaulUithol** [comment]
Also still some older tools around we'd need to choose what to do with, such as http://visualtags.hotosm.org/

**Comment by TylerRadford** [comment]
I would add OSM Analytics to the first category

**Comment by smit1678** [comment]
Nice catch @TylerRadford, thanks.

Thanks @PaulUithol. Yeah I think there are many collaborations or other projects where we could be supportive more. In terms of this exercise, I think we just want to list out the major active ones where we need to have branding considerations. 

@PaulUithol Is visualtags.hotosm.org used much? 

Updating the list below here: 

#### Core tools/sites:

- hotosm.org
- Country offices/intiatives
  - HOT ID
- Tasking Manager
- Export Tool
- Training Center
- OSM Analytics

#### HOT-led/coordinated Projects or Initiatives:

- OpenAerialMap
- LearnOSM
- Field Papers (to be HOT-led)
- Collaborations
  - Ramani Huria
  - Missing Maps
  - MapUganda
  - YouthMappers

**Comment by RebeccaFirthy** [comment]
Looks good to me

**Comment by xamanu** [comment]
Looks good. Just a little thing, I'd split up in three types:

1. HOT core tools and sites
2. HOT-led/coordinated Projects or Initiatives
3. Collaborations

Probably the three categories are for each one going to talk a different visual language.

Cheers, Felix

**Comment by vannicer** [comment]
@smit1678 makes sense to me. The order within the categories isn't significant, correct?

**Comment by PaulUithol** [comment]
@smit1678 not sure how much visualtags is used, not much I'd guess. David should be able to pull stats per domain from somewhere. Might discover some more old ones in the process (that we could either just leave be, formally deprecate, or apply new styling to as well?).
